fly.toml

```diff
@@ -0,0 +1,48 @@
+# https://fly.io/docs/reference/configuration
+
+app = "dagger-registry-2023-01-23"
+kill_signal = "SIGINT"
+# Wait these many seconds for existing connections to drain before hard killing
+kill_timeout = 30
+processes = []
+
+# There is no build config since this will always be deployed with a specific
+# --image flag. This is currently a pipeline concern. FTW, the app image has
+# the build run embedded in /BUILD_URL file.
+# [build]
+
+[env]
+  PORT = "8080"
+
+[deploy]
+  strategy = "bluegreen"
+
+[experimental]
+  auto_rollback = true
+  cmd = ["-repo", "dagger"]
+
+[[services]]
+  http_checks = []
+  internal_port = 8080
+  processes = ["app"]
+  protocol = "tcp"
+  script_checks = []
+  [services.concurrency]
+    hard_limit = 25
```

Comment:
Yes. New requests will not finish the handshake and most likely result in timeouts.

Increasing this to 1000 sounds reasonable to me. I wouldn't go higher since I don't expect a single shared CPU to be able to service more than this and still offer a good QoS.

If we find ourselves hitting this limit, we should consider setting up autoscaling. FWIW https://community.fly.io/t/connection-count-limits/99 .